content/about/duje/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: About - Duje Schröder
+weight: 10
+---
+
+<div class="text-center mb-5">
+    <img
+        src="https://avatars.githubusercontent.com/u/23352538?v=4"
+        width="150"
+        class="rounded-circle mt-3"
+    />
+    <h3 class="m-3">Duje Schröder</h3>
+    <p>Assistant Coordinator</p>
+</div>
+
+<div class="text-center">
+   _,--',     _,_._.--._______
+ .--.--';_'-. ',_";_      _.,-'
+.'--'.  _.'    {`'-;_ .-...'
+      '-:_      )  / `' '=.
+        ) >     {_/     /.)..
+        |/               `..; .'
+</div>
+
+
+End? No, the journey doesn't end here. MapLibre is another path. One that we
+all must take. On 2020-12-08 22:42:56 UTC MapLibre restarted mapping with its
+mission to provide freedom and independence. Eventually, this is what everybody
+is seeking, from a little project to enterprise endeavours. I am Duje and I am
+assisting as a Coordinator to help maintain what the community has built so
+carefully which means to me:
+
+- Assure MapLibre's fundraising and a healthy Sponsorship Program to keep
+  MapLibre on the map
+- Listen to and discuss with the vivid community which corner to turn next
+- Make MapLibre seen and accessible to everyone
+
+To sidewalk talks where everyone feels comfortable in the MapLibre world.
+
+Gladly find me in my alley: <span><a
```

Comment:
I'd rephrase this, as well. Something like "You can always find me on Slack". This phrase is a bit awkward in English.

Reply:
@duje I will leave this one up to you. Let's merge it like this and if you want to update it, feel free to submit a follow-up pull request.

Reply:
Let it be a nice alley that one would find in Dubrovnik, Lisbon or Zurich 🙂